app/views/create_user.erb

```diff
@@ -0,0 +1,27 @@
+<form action="/create_user" method="post">
+  <label for='name'>
+		Name:
+		<input type="text" name="name">
+	</label>
+  <br />
```

Comment:
avoid layout concerns in your HTML.  HTML is for _semantic_ structure only - layout is handled by CSS.  Generally, `<br />` tags are a design smell.